### ISSUE TITLE
Anyone in linux (ubuntu) in my case, who also didn't get it automatically disabled?

### ISSUE THREAD

**Issue by mkrier** [deanoemcke/thegreatsuspender#1325]
* Extension version: The Great Suspender
7.1.8
 * Browser name & version: Chrome Version 88.0.4324.96 (Official Build) (64-bit)
 * Operating system & version: Ubuntu 18.04.5 LTS

My Great Suspender never got automatically disabled like all the articles said... is that because I'm running linux?  I'm guessing the answer is no... so then... why not?  Anyone else experience the same thing?  thanks.
Oh and should I disable it? It's still not in the google store, but again, it didn't get disabled so maybe I'm running a safe version?

**Comment by chromex86** [comment]
Hi there, I'm running Ubuntu 16.04 LTS with Chromium 71.0.3578.98 and it got automatically disabled 2 days ago just like in other reports.

Extension version same as yours.

**Comment by mkrier** [comment]
Damn.. what's special about me that makes it still enabled?  And I guess I should probably disable it right...

**Comment by sarog** [comment]
> Damn.. what's special about me that makes it still enabled? And I guess I should probably disable it right...

In case you still haven't disabled it, let me make it super clear for you: DISABLE THE EXTENSION NOW!

**Comment by skyrail01** [comment]
Windows 10 - Chrome 88.0.4324.150 (Build officiel) (64 bits) - TGS 7.1.8.
Not automatically suspended. Still functionnal. Time to find an alternative. Not sure the Marvelous version or going back to the last release 7.1.6 is safe.
Nevertheless, is there a way to get the TGS session management history back in the Marvelous version?

**Comment by mkrier** [comment]
> Windows 10 - Chrome 88.0.4324.150 (Build officiel) (64 bits) - TGS 7.1.8.
> Not automatically suspended. Still functionnal. Time to find an alternative. Not sure the Marvelous version or going back to the last release 7.1.6 is safe.
> Nevertheless, is there a way to get the TGS session management history back in the Marvelous version?

Yes I would like to know too. Maybe start a new thread for that question specifically?

**Comment by PatrickLearn** [comment]
> Damn.. what's special about me that makes it still enabled? And I guess I should probably disable it right...

Are you using ungoogled-chromium?  That could be why, but it seems too obvious. You might also just be using chromium instead of google chrome, which I do not know if it would be disabled on there.

**Comment by uunnxx** [comment]
> Are you using ungoogled-chromium? That could be why, but it seems too obvious. You might also just be using chromium instead of google chrome, which I do not know if it would be disabled on there.

No, in chromium I also get it disabled automatically about month ago.
